cmd/rook/ceph/cleanup.go

```diff
@@ -87,3 +108,38 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func startSubVolumeGroupsCleanUp(cmd *cobra.Command, args []string) error {
+	rook.SetLogLevel()
+	rook.LogStartupInfo(cleanUpSubVolumeGroupsCmd.Flags())
+
+	ctx := cmd.Context()
+	context := createContext()
+	namespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
+	clusterInfo := client.AdminClusterInfo(ctx, namespace, "")
+
+	fsName := os.Getenv(opcontroller.CephFSVolumeNameEnv)
+	if fsName == "" {
+		return fmt.Errorf("cephfs volume name is not available in the pod environment variables")
+	}
+	subVolumeGroupName := os.Getenv(opcontroller.CephFSSubVolumeGroupNameEnv)
+	if subVolumeGroupName == "" {
+		return fmt.Errorf("cephfs SubVolumeGroup name is not available in the pod environment variables")
+	}
+	csiNamespace := os.Getenv(opcontroller.CephFSNamesaceEnv)
+	if csiNamespace == "" {
+		return fmt.Errorf("cephfs SubVolumeGroup namespace name is not available in the pod environment variables")
+	}
+
+	poolName := os.Getenv(opcontroller.CephFSMetaDataPoolNameEnv)
+	if poolName == "" {
+		return fmt.Errorf("cephfs metadata pool name is not available in the pod environment variables")
+	}
+
+	err := cleanup.StartSubVolumeGroupCleanup(context, clusterInfo, fsName, subVolumeGroupName, poolName, csiNamespace)
```

Comment:
"Start" as a method prefix implies that it will be done asynchronously, which isn't the case for this command.
```suggestion
	err := cleanup.SubVolumeGroupCleanup(context, clusterInfo, fsName, subVolumeGroupName, poolName, csiNamespace)
```